yarn.lock

```diff
@@ -1708,6 +1708,11 @@
     open "^8.0.5"
     ora "^5.4.0"
 
+"@contentful/app-sdk@^4.15.0":
+  version "4.15.0"
+  resolved "https://registry.yarnpkg.com/@contentful/app-sdk/-/app-sdk-4.15.0.tgz#ce35116168b59f3c1e04ac4f869c18a4bc874f3e"
+  integrity sha512-dcawQN7fAOVmKsrAykpJB8O6hkBH2b9jWAn3/Ts/h16xUKs5R7409HPt3akB1pBKiifJYRMuTZAYC6xkj17+nw==
+
 "@contentful/app-sdk@^4.2.0", "@contentful/app-sdk@^4.3.0", "@contentful/app-sdk@^4.6.0":
```

Comment:
Should we try to get everything to the same version and dedupe App SDK?

Reply:
Good idea, done!

Reply:
> * Does this change the interface for the users?
`useEntityInfo` has now more strict types, but it's not like we supported anything for other string values than `Entry` and `Asset`. 
> * Do we have to bump the peer dependencies? This normally requires a breaking change as we drop support for old versions
I updated the app-sdk devDependencies for all relevant packages and thought it would be odd to have a lower peerDependency requirement. But technically you are right, it is not required. Would you prefer to revert that?
> * Do we want to release a new version?
Yes, I want to release a version which is then used in experience packages to align the versions of app-sdk used in this repo with the one in experience packages.